src/components/formControls/textarea/index.tsx

```diff
@@ -8,17 +8,25 @@ import { StyledWrapper, StyledArea } from './style'
 export interface Props {
   id?: string
   value?: string
+  defaultValue?: string
+  placeholder?: string
   disabled?: boolean
   onChange?: (event: React.ChangeEvent<HTMLTextAreaElement>) => void
 }
 
 export default class TextArea extends React.PureComponent<Props, {}> {
   render () {
-    const { id, onChange, value, disabled } = this.props
+    const { id, onChange, value, defaultValue, placeholder, disabled } = this.props
 
     return (
       <StyledWrapper id={id}>
-        <StyledArea onChange={onChange} disabled={disabled} value={value} />
+        <StyledArea
+          onChange={onChange}
+          disabled={disabled}
+          value={value}
+          placeholder={placeholder}
+          defaultValue={defaultValue}
```

Comment:
ya React has it, more in https://reactjs.org/docs/uncontrolled-components.html#default-values. I ended up not using it here but seems good to have since it is valid per React code.